depends/packages/libxcb.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=98d9ab05b636dd088603b64229dd1ab2d2cc02ab807892e107d674f9c
 $(package)_dependencies=xcb_proto libXau
 
 define $(package)_set_vars
-$(package)_config_opts=--disable-static
+$(package)_config_opts=--disable-static --disable-build-docs --without-doxygen
```

Comment:
This is a candidate for another issue... It seems that we build all of the default XCB Extensions, but the `-qt-xcb` option we supply to QT might cause it to build them as well. (source: https://doc.qt.io/qt-5.9/linux-requirements.html)

Reply:
Opened an issue to track that: https://github.com/bitcoin/bitcoin/issues/16447